lib/storage/__mocks__/index.native.js

```diff
@@ -0,0 +1,3 @@
+import * as Storage from '@react-native-async-storage/async-storage/jest/async-storage-mock';
```

Comment:
In tests storage is mocked by by the AsyncStorage's mock
Currently it matches perfectly with our storage interface as the interface is based on the AsyncStorage api